flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/inference/OperatorBindingCallContext.java

```diff
@@ -59,7 +59,7 @@ public OperatorBindingCallContext(
                 sqlOperatorBinding.getOperator().getNameAsId().toString(),
                 sqlOperatorBinding.getGroupCount() > 0);
 
-        this.binding = new OperatorBindingDecorator(sqlOperatorBinding);
+        this.binding = sqlOperatorBinding;
```

Comment:
nit: sqlOperatorBinding -> binding

Reply:
Updated both  `OperatorBindingcallContext` and `CallBindingCallContext`